lib/reporters.js

```diff
@@ -21,16 +21,17 @@ NullReporter.prototype.report = function(series, onSuccess, onError) {
 // DataDogReporter
 //
 
-function DataDogReporter(apiKey, appKey, agent) {
+function DataDogReporter(apiKey, appKey, agent, apiHost) {
     apiKey = apiKey || process.env.DATADOG_API_KEY;
     appKey = appKey || process.env.DATADOG_APP_KEY;
+    apiHost = apiHost || process.env.DATADOG_API_HOST;
```

Comment:
Wouldn't `apiHost` need a default value as it's an optional parameter in the init function.

Reply:
dogapi is already taking care of api_host being undefined. Should be fine